Rakefile

```diff
@@ -83,7 +83,7 @@ task 'dlls' do
   env += 'EMBED_ZLIB=true '
   env += 'BUILDDIR=/tmp '
   env += "V=#{verbose} "
-  out = '/tmp/libs/opt/grpc-1.dll'
```

Comment:
Instead of moving the whole Rakefile into a template, wouldn't it make a bit more sense to have that single string into a loadable .rb file that the Rakefile would load ?